model/configuration.py

```diff
@@ -58,25 +63,47 @@ class ExternalIntegrationLink(Base, HasFullTableCache):
     )
     purpose = Column(Unicode, index=True)
 
-    types = [COVERS, BOOKS]
+    mirror_settings = [
+        {
+            'key': COVERS_KEY,
+            'type': COVERS,
+            'description_type': 'cover images',
+            'label': 'Covers Mirror'
+        },
+        {
+            'key': OPEN_ACCESS_BOOKS_KEY,
+            'type': OPEN_ACCESS_BOOKS,
+            'description_type': 'free books',
+            'label': 'Open Access Books Mirror'
+        },
+        {
+            'key': PROTECTED_ACCESS_BOOKS_KEY,
+            'type': PROTECTED_ACCESS_BOOKS,
+            'description_type': 'self-hosted books',
```

Comment:
Open access books are also self-hosted, so how about calling this "self-hosted, commercially licensed books" or similar?

Reply:
I changed the `description_type` as requested